statesman-engine/src/main/java/io/appform/statesman/engine/handlebars/HandleBarsHelperRegistry.java

```diff
@@ -354,6 +356,30 @@ private void registerAdd() {
         });
     }
 
+    private void registerComputeDays() {
```

Comment:
Add test case in HandlebarsServiceTest

Reply:
Since I use System.currentTimestamp, I have to use power mockito or some other hack. Hence didn't add.

Reply:
Write a delegate that accepts epoch and test that. From this function call the other function with `System.currentTimestamp()`

Reply:
done. Used Clock instance to get time